src/page-modules/assistant/layout.tsx

```diff
@@ -187,6 +199,32 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
                     data={transportModeFilter}
                     onChange={onTransportFilterChanged}
                   />
+
+                  <div>
+                    <Typo.p
```

Comment:
Maybe this should have a more semantically correct element like h3 🤔

Reply:
Are you sure? Both the filter title and the search assistant title are both <Typo.p textType="body__primary--bold" className={style.heading}>.

Reply:
Might make sense to change those to be `h3` as well possibly.

Reply:
Yes, that might make sense. In any case, I think that all of these titles should have the same font.